src/main/java/net/minecraftforge/event/world/ChunkWatchEvent.java

```diff
@@ -57,6 +64,15 @@ public EntityPlayerMP getPlayer()
         return player;
     }
 
+    /**
+     * The world of the chunk.
+     * @return
+     */
+    public World getWorld()
```

Comment:
You should probably mark this as `@Nullable` until the original constructors get removed, since `world` may not be initialised.

Reply:
At the moment it sets the world to the player's world if the old constructors are used, so it won't be null. But you're probably right, setting it to null instead might be better because then the event handlers can know whether the world is actually correct or not in case something uses the old constructors.
I'll have another look at this tomorrow

Reply:
To be completely safe you can mark it nullable with a comment saying to remove the nullable annotation when the old constructor is removed.